### DISCUSSION TITLE
Wake up and shut down commands not working

### DISCUSSION THREAD

**Discussion by kingkaden123** [AlexGustafsson/homebridge-wol#210] [Questions] [answered]
Hello! I'm getting the error "werr_call_not_implemented" when trying to send a start or shut down command. I tried searching around, but I can't find a way to fix it. Thanks in advance for any help you can provide!

Here are my logs:
![Screenshot 2022-07-18 124551](https://user-images.githubusercontent.com/107531062/179604688-a79839e5-566c-4c60-b244-eb79cd8d72e2.jpg)

**Comment by AlexGustafsson** [comment, marked as answer]
It seems that if your computer doesn't support the "call" (shutdown), which shouldn't happen during normal circumstances. Could it be that there's some configuration necessary on your target PC?

https://www.reddit.com/r/homeassistant/comments/epn9w0/question_about_rpc_shutdown_and_windows_10/

**Reply by kingkaden123** [reply]
Yes, I think you are right. I was able to fix this by installing magic packet server. 

This: https://apps.somenkov.ru/magic-packet/server/

Now everything shuts down perfectly. Thanks so much for the help!
